packages/strapi-design-system/src/v2/SimpleMenu/Menu.tsx

```diff
@@ -25,16 +25,18 @@ const MenuRoot = DropdownMenu.Root;
  * MenuTrigger
  * -----------------------------------------------------------------------------------------------*/
 
-interface TriggerProps extends ButtonProps {}
+interface TriggerProps extends ButtonProps {
+  noIcon?: boolean;
```

Comment:
I don't know what the others think, but I am not a big fan of this prop, because it doesn't scale well.

What do you think about passing in `endIcon` instead? It could be set to `<CarretDown width={`${6 / 16}rem`} height={`${4 / 16}rem`} aria-hidden />` by default, and we allow overwriting it from outside?

Reply:
I understand your point, and what do you suggest to pass from the CrumbSimpleMenu component as value for the endIcon? null?

Reply:
Yes :)